src/include/duckdb/parser/expression/between_expression.hpp

```diff
@@ -17,6 +17,7 @@ class BetweenExpression : public ParsedExpression {
 	static constexpr const ExpressionClass TYPE = ExpressionClass::BETWEEN;
 
 public:
+	BetweenExpression();
```

Comment:
See https://github.com/Mytherin/duckdb/pull/4, basically I think it would be better to move this to private + using directly `auto x = unique_ptr<BetweenExpression>(new BetweenExpression());` and keeping this an implementation detail as private constructor.

Reply:
Thanks - good idea